Add perimeter

andyjenkinson opened the pull request:
Given area is included in core and the effort to include perimeter is identical, it makes sense to include it.

m-mohr commented:
This needs a changelog entry, the merge conflicts need to be resolved and I'm a bit curious about the maximum number, but generally this looks good.

andyjenkinson commented:
Fixed the merge conflict: description of area from 'should' to 'must' was updated in the meantime, but I'd already made the same change. Added changelog entry.

The perimeter limit is calculated as approximately the perimeter (in metres) you'd expect from a square field of the (huge) size that the maximum area is currently set to (in hectares), rounded to a nicer number. I figure if we don't know what a real maximum field is, it makes sense just to make them consistent with each other. Perhaps you can check my maths!

Will you not do a changelog when a release is made?

m-mohr commented:
Thank you.

Makes sense and I got to the same number.

We could change to PRs only and auto-generate from them, but I didn't follow that at the beginning and pushed directly to main before the sprint. I'm also just used to keep the changelog consistent at any time...
